### ISSUE TITLE
Bugfix: Bomberman Hero Crash

### ISSUE THREAD

**Issue by GoogleCodeExporter** [facelift/mupen64gc#226]
```
What steps will reproduce the problem?
1. Go to the fourth planet, Mazone, the second area, and then Snowland.
2. Go past the second snowman.

What is the expected output? What do you see instead?
The game freezes, without a code dump, and the Wii has to be hard-resetted.

What version of the product are you using? On which system and loader?
Wii64 1.1, Wii System Menu 4.2, HBC 1.0.6

Please provide any additional information below.
I've included my save and the save-state straight into the level.
```

Original issue reported on code.google.com by `DaX...@gmail.com` on 3 Mar 2010 at 10:12

Attachments:
- [BOMBERMAN HERO(U).eep](https://storage.googleapis.com/google-code-attachments/mupen64gc/issue-226/comment-0/BOMBERMAN HERO%28U%29.eep)
- [BOMBERMAN HERO(U).st0](https://storage.googleapis.com/google-code-attachments/mupen64gc/issue-226/comment-0/BOMBERMAN HERO%28U%29.st0)

**Comment by GoogleCodeExporter** [comment]
Original comment by `emuki...@gmail.com` on 11 May 2010 at 12:31
- Changed title: **Bugfix: Bomberman Hero Crash**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in > v1.1
```

Original comment by `emuki...@gmail.com` on 9 Oct 2012 at 11:44
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
But I have v1.1 and it still crashes sooo... what do I do?
```

Original comment by `ljinkk...@gmail.com` on 20 Dec 2012 at 11:18
